app/posts/apply-for-teacher-training/2021-07-14-selecting-references.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Selecting references
+description: Letting candidates choose which references to submit with their application
+date: 2021-07-14
+---
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+
+Previously candidates were unable to choose which 2 references would be sent with their application to providers. They would request several references and whichever 2 they received first would be sent with their application.
+
+We decided to build a feature that allows candidates to select which 2 references they want to send to providers.
+
+{{ appScreenshots({
+  hideContents: true,
+  items: [{
+    text: "Select 2 references link on task list",
+    img: {
+      src: "Select-2-references-link.png"
```

Comment:
Use lowercase filenames for images (aka rename the image files, and update the references in this markdown file)

```suggestion
      src: "select-2-references-link.png"
```